lottie/src/main/java/com/airbnb/lottie/utils/Utils.java

```diff
@@ -67,6 +69,15 @@ protected float[] initialValue() {
 
   private static final float INV_SQRT_2 = (float) (Math.sqrt(2) / 2.0);
 
+  private static final ColorFilter lumaColorFilter = new ColorMatrixColorFilter(
```

Comment:
Thanks for looking into this. What is the source for the values in this matrix?

Reply:
@alexzhirkevich You resolved this comment but could you provide the source for thsi?

Reply:
It doesn't work well anyway as you can remember. I'll close this one until a better approach appears